lms/envs/common.py

```diff
@@ -2170,7 +2184,7 @@ def _make_locale_paths(settings):
     'openedx.core.djangoapps.dark_lang',
 
     # Microsite configuration
-    'microsite_configuration',
+    'microsite_configuration.apps.MicrositeConfigurationConfig',
```

Comment:
I'm confused as to why we need the class name in INSTALLED_APPS now and not just the app name?

Reply:
We discussed this recently in another PR.  The summary seems to be that the main developer of the new Django application initialization system in 1.7 strongly believes that the configuration should be as explicit as possible, but a lot of other people agree with you that it's needless clutter.  He compromised by adding the `default_app_config` attribute to permit the simpler `INSTALLED_APPS` entries, but adding a note to the documentation saying that using it is strongly discouraged.

Reply:
A few of us had this discussion on another PR. While you can specify a `default_app_config` in the `__init.py__` of an app, the Django best practice seems to be *not* defining it and specifying the app's AppConfig directly in INSTALLED_APPS. I mildly preferred the shorter name in INSTALLED_APPS - but we decided to go with this convention instead.

Reply:
The more you know.